### ISSUE TITLE
Selective Generation

### ISSUE THREAD

**Issue by Drok00** [mephitrpg/rpg-cards#23]
If there was an option to selectively generate only a few cards of a large deck it would be useful if you only needed a few cards to print. Printing an entire deck when you only needed to make edits to a dozen or so can be rather troublesome.

**Comment by crobi** [comment]
I agree. The workflow I had in mind is that you have your deck with all cards, and then you delete some of them before you print.

The cards now have tags (e.g., you could add the "mage" tag to all spells that can be cast by a mage), and there is a crude interface to manipulate cards according to their data (the "Map/Filter" button). Currently, the interface is not more than a javascript console, but I am planning to make it more user friendly, so that you could for example delete all cards without a given tag.

If you have a better idea how to select the cards to print, I'm open to suggestions!

**Comment by Drok00** [comment]
I see two fairly simple methods. Regarding tags, a drop-down menu listing all the tags for a simple sort.  Without tags I would imagine a grid selection with check boxes listing every card. Both methods become unwieldy with very large decks, but I'm not sure of a better solution.

**Comment by crobi** [comment]
I agree that the grid selection becomes unwieldy for large decks (the deck with all D&D 5e spells contains over 350 cards). I didn't understand how you would use the tags though, or why using the tags would be unwieldy.

If your use case is printing a small random subset of the deck where the cards in the subset have no common tag (maybe because you corrected a spelling mistake and want to reprint them), then my approach won't work for you. In that case, some system where you mark individual cards and then only the marked cards get generated would be better.

**Comment by DesktopMan** [comment]
"Generate this card only" would be a great start as I keep a bunch of cards in the deck and print as they are needed.
